codec-stomp/src/test/java/io/netty/handler/codec/stomp/StompSubframeDecoderTest.java

```diff
@@ -155,4 +160,38 @@ public void testMultipleFramesDecoding() {
 
         assertNull(channel.readInbound());
     }
+
+    @Test
+    public void testValidateHeadersDecodingDisabled() {
+        ByteBuf invalidIncoming = Unpooled.buffer();
+        invalidIncoming.writeBytes(StompTestConstants.FRAME_WITH_INVALID_HEADER.getBytes());
+        channel.writeInbound(invalidIncoming);
+
+        StompHeadersSubframe frame = channel.readInbound();
+        assertNotNull(frame);
+        assertEquals(StompCommand.SEND, frame.command());
+        assertTrue(frame.headers().contains("destination"));
+        assertTrue(frame.headers().contains("content-type"));
+        assertFalse(frame.headers().contains("current-time"));
+
+        StompContentSubframe content = channel.readInbound();
+        String s = content.content().toString(CharsetUtil.UTF_8);
+        assertEquals("some body", s);
+        content.release();
```

Comment:
call `channel.finish()` and validate the return value

Reply:
@After
    public void teardown() throws Exception {
        assertFalse(channel.finish());
    }

Reply:
ah sorry missed this 👍